package-lock.json

```diff
@@ -2784,8 +2784,7 @@
 		"node_modules/@types/lodash": {
 			"version": "4.14.201",
 			"resolved": "https://registry.npmjs.org/@types/lodash/-/lodash-4.14.201.tgz",
-			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ==",
-			"dev": true
+			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ=="
```

Comment:
I'm not sure why the ``"dev": true`` line following this one was removed but I don't think it should be part of this PR. If you think this is an issue then please put one on GitHub so it can be addressed separately.

Reply:
We will update the PR to remove changes to package-lock.json

Reply:
We updated the PR with your feedback :) Let us know if you have other feedback!